Packs/FortiGate/Integrations/FortiGate/FortiGate.py

```diff
@@ -1,93 +1,1493 @@
+            format_fields (list[str], optional): The fields to format.
+                Defaults to None.
+
+        Returns:
+            dict[str, Any]: The API response.
+        """
+        return self._http_request(
+            method="GET",
+            url_suffix=urljoin(self.ADDRESS_IPV4_ENDPOINT, name) if name else self.ADDRESS_IPV4_ENDPOINT,
+            params=remove_empty_elements(
+                {
+                    "vdom": vdom,
+                    "filter": self._get_filter(filter_field, filter_value),
+                    "format": self._get_format(format_fields),
+                }
+            ),
+            error_handler=Client._error_handler,
+        )
+
+    def create_firewall_address_ipv4(
+        self,
+        name: str,
+        type_: str,
+        vdom: str | None = DEFAULT_VDOM,
+        comment: str | None = None,
+        associated_interface: str | None = None,
+        address: str | None = None,
+        mask: str | None = None,
+        allow_routing: str | None = None,
+        start_ip: str | None = None,
+        end_ip: str | None = None,
+        fqdn: str | None = None,
+        country: str | None = None,
+        mac_addresses: list[str] | None = None,
+    ) -> dict[str, Any]:
+        """Create a IPv4 address.
+
+        Args:
+            name (str): The name of the address to create.
+            type_ (str): The type of the IPv4 address to create.
+            vdom (str, optional): The VDOM to use.
+                Defaults to VDOM_DEFAULT.
+            comment (str | None, optional): A comment for the address.
+                Defaults to None.
+            associated_interface (str | None, optional): Network interface associated with address.
+                Defaults to None.
+            address (str | None, optional): The IP address.
+                Defaults to None.
+            mask (str | None, optional): The subnet mask of the address.
+                Defaults to None.
+            allow_routing (str | None, optional): Enable/disable use of this address in the static route configuration.
+                Defaults to None.
+            start_ip (str | None, optional): First IP address (inclusive) in the range for the address.
+                Defaults to None.
+            end_ip (str | None, optional): Final IP address (inclusive) in the range for the address.
+                Defaults to None.
+            fqdn (str | None, optional): Fully Qualified Domain Name address.
+                Defaults to None.
+            country (str | None, optional): IP addresses associated to a specific country.
+                Input must be according to the two-letter counter codes, for example: `IL`.
+                Defaults to None.
+            mac_addresses (list[str] | None, optional): list of MAC addresses.
+                Defaults to None.
+
+        Returns:
+            dict[str, Any]: The API response.
+        """
+        return self._http_request(
```

Comment:
`does_path_exists` validates whether the endpoint exists, I'm not sure why this would be needed.
The payload has been expanded to allow more type of addresses to be created, if the arguments will be passed, then the result will be the same as before.

Reply:
An error will be returned with or without the use of `does_path_exist`, the only change here is the message.
For update commands, it'll show a `404` for not found, because it doesn't exist.
And for create commands, it'll show a `500`  for server error, because it already exists.